supertokens_python/recipe/emailverification/interfaces.py

```diff
@@ -161,24 +165,55 @@ def __init__(self):
 
     @abstractmethod
     async def email_verify_post(
-        self, token: str, api_options: APIOptions, user_context: Dict[str, Any]
+        self,
+        token: str,
+        api_options: APIOptions,
+        session: Optional[SessionContainer],
+        user_context: Dict[str, Any],
     ) -> Union[
         EmailVerifyPostOkResult, EmailVerifyPostInvalidTokenError, GeneralErrorResponse
     ]:
         pass
 
     @abstractmethod
     async def is_email_verified_get(
-        self, api_options: APIOptions, user_context: Dict[str, Any]
+        self,
+        api_options: APIOptions,
+        session: Optional[SessionContainer],
```

Comment:
session is not optional and should be after self

Reply:
done